engine/verification/utils/unittest/helper.go

```diff
@@ -591,10 +595,10 @@ func withConsumers(t *testing.T,
 	}
 
 	// verifies memory resources are cleaned up all over pipeline
-	assert.True(t, verNode.BlockConsumer.Size() == 0)
-	assert.True(t, verNode.ChunkConsumer.Size() == 0)
-	assert.True(t, verNode.ChunkStatuses.Size() == 0)
-	assert.True(t, verNode.ChunkRequests.Size() == 0)
+	assert.Equal(t, verNode.BlockConsumer.Size(), uint(0))
+	assert.Equal(t, verNode.ChunkConsumer.Size(), uint(0))
+	assert.Equal(t, verNode.ChunkStatuses.Size(), uint(0))
+	assert.Equal(t, verNode.ChunkRequests.Size(), uint(0))
```

Comment:
maybe invert the order of parameters. The first one is usually the expected value.

Reply:
I updated to use `assert.Zero` https://github.com/onflow/flow-go/pull/4498/commits/6b9cce5e8250e4f2f1f6f1b07f6f3f450a8f28f5